client/landing/stepper/stores.ts

```diff
@@ -22,3 +23,5 @@ export const USER_STORE = User.register( {
 } );
 
 export const AUTOMATED_ELIGIBILITY_STORE = AutomatedTransferEligibility.register();
+
+export const BLOCK_RECIPES_STORE = BlockRecipes.register();
```

Comment:
Hmm, since the recipes are used only in the design picker, I wonder if it is much simpler to just fetch them using a react-query hook (`useGeneratedDesigns()`), similar to the existing logic in `useThemeDesignsQuery()`?